packages/url_launcher/url_launcher_platform_interface/lib/url_launcher_platform_interface.dart

```diff
@@ -38,6 +39,9 @@ abstract class UrlLauncherPlatform extends PlatformInterface {
     _instance = instance;
   }
 
+  /// The delegate used by the Link widget to build itself.
+  LinkDelegate get linkDelegate;
```

Comment:
This is defined as:

```dart
  @override
  LinkDelegate linkDelegate = (LinkInfo linkInfo) => WebLinkDelegate(linkInfo);
```

In the web implementation, how can you override a getter with the function above? Should the signature here be more explicit?